Observe Flake8 on setting to skip publish when live pages submitted.

mattlinares opened the pull request.

zerolab commented on wagtail_localize/operations.py:
```diff
@@ -72,8 +72,8 @@ def create_translations(self, instance, include_related_objects=True):
-            # Determine whether to publish the translation.
-            if getattr(settings, "WAGTAILLOCALIZE_SYNC_LIVE_STATUS_ON_TRANSLATE", True):
+            publish_on_translate_setting = getattr(settings, "WAGTAILLOCALIZE_SYNC_LIVE_STATUS_ON_TRANSLATE", True)
+            if publish_on_translate_setting is True:
```
this is an unnecessary change

mattlinares commented:
@zerolab I changed it as I'm finding that it doesn't work because `if getattr(settings, "WAGTAILLOCALIZE_SYNC_LIVE_STATUS_ON_TRANSLATE", True):` is always `True`.

I added a breakpoint and print statement, see the output:

```
> /root/.cache/pypoetry/virtualenvs/.venv/lib/python3.10/site-packages/wagtail_localize/operations.py(77)create_translations()
-> print(getattr(settings, "WAGTAILLOCALIZE_SYNC_LIVE_STATUS_ON_TRANSLATE", True))
(Pdb) s
False
> /root/.cache/pypoetry/virtualenvs/.venv/lib/python3.10/site-packages/wagtail_localize/operations.py(78)create_translations()
-> if getattr(settings, "WAGTAILLOCALIZE_SYNC_LIVE_STATUS_ON_TRANSLATE", True):
(Pdb) s
> /root/.cache/pypoetry/virtualenvs/.venv/lib/python3.10/site-packages/wagtail_localize/operations.py(79)create_translations()
-> publish = getattr(instance, "live", True)
(Pdb) s
> /root/.cache/pypoetry/virtualenvs/.venv/lib/python3.10/site-packages/wagtail_localize/operations.py(83)create_translations()
-> try:
(Pdb) publish
True
(Pdb)
```

with:
```
# Determine whether to publish the translation.
            breakpoint()
            print(getattr(settings, "WAGTAILLOCALIZE_SYNC_LIVE_STATUS_ON_TRANSLATE", True))
            if getattr(settings, "WAGTAILLOCALIZE_SYNC_LIVE_STATUS_ON_TRANSLATE", True):
                publish = getattr(instance, "live", True)
            else:
                publish = False
```

zerolab commented:
Hey @mattlinares I have some time on Friday where I planned to check on this, and make a new release.

Did you close it as you don't think it is relevant, or...?

zerolab commented:
Added only the test in #677 and it is all green so will merge that (with attribution)

mattlinares commented:
Thanks @zerolab
